bulma-rtl.sass

```diff
@@ -0,0 +1,10 @@
+/*! bulma.io v0.4.0 | MIT License | github.com/jgthms/bulma */
+@charset "utf-8"
+
+$right: left
+$left: right
+
+html
+  direction: rtl
+
+@import "bulma"
```

Comment:
your provided approach is nearly similar to what I did. the only difference between using html[dir=rtl] and using $right $left variables is in result and the implementation of both is the same. the result of my approach is 2 separated css files after build (bulma.css for LTR and bulma-rtl.css for RTL that library users just need to include one of these in the html page) and your approach will result in just 1 bulma.css which will have a bigger size (as you mentioned too) and includes both styles for RTL and LTR. actually my separation happens in SASS to CSS transpilation process and your separation happens in rendering time in browser.

I don't know if you are the one who can accept my PR or not (since you are not a member or contributor). so for now I accept your solution as an alternative and if the maintainers want me to edit my changes with this solution then I will update my PR. please let me know if you are the maintainer then I will apply your solution to the PR otherwise we can wait for a decision from maintainers on this.

finally in my opinion both approaches (mine and yours) have some drawbacks and I can't prefer one to another and I can't say one of them is pretty and another is not. I think all of these are dirty solutions until we have something like direction: auto; in css. we already have a dir=auto attribute in html and it's almost perfect but it's only applicable on divs/inputs and textual contents but doesn't affect in layout. by the way here I can reject your thoughts on: 

> Most RTL pages on the web will have <html dir="rtl">, right?

I can provide many examples that dir="auto" is the most used value for dir. so your approach is not usable in these cases.